src/mvs/CMakeLists.txt

```diff
@@ -20,6 +20,10 @@ if(CUDA_ENABLED)
         # Workaround for Visual Studio compiler error,
         # due problems with compiling too complex code.
         string(REPLACE "/Zo" "" CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
+
+        # Avoid pulling in too many header files through <windows.h>
+        add_definitions(-DWIN32_LEAN_AND_MEAN)
```

Comment:
I have only got CUDA to work with MSVC 2013, so it only shows up there. It probably also depends on which Win SDK you have installed.